sql/sys_vars.cc

```diff
@@ -1513,7 +1513,9 @@ static Sys_var_bit Sys_log_slow_admin_statements(
        "to the slow log if it is open.  Resets or sets the option 'admin' in "
        "log_slow_disabled_statements",
        SESSION_VAR(log_slow_disabled_statements),
-       CMD_LINE(OPT_ARG), REVERSE(LOG_SLOW_DISABLE_ADMIN), DEFAULT(TRUE));
+       CMD_LINE(OPT_ARG), REVERSE(LOG_SLOW_DISABLE_ADMIN), DEFAULT(TRUE),
+       0, NOT_IN_BINLOG, ON_CHECK(0), ON_UPDATE(0),
+       DEPRECATED("'@@log_slow_filter=REPLACE(@@log_slow_filter,\"admin\",\"\")'")); // since 10.11.2
 
```

Comment:
**first thought**: please use SQL standard single quotes for strings, not MariaDB extension double quotes.
**second thought**: it's a rather complex line for a replacement, may be simply
```
'@@log_slow_admin_statements' is deprecated and will be removed in a future release. Please use '@@log_slow_filter' instead
```
?

Reply:
original thought, it wasn't a one to one deprecation variable.
better though consistent with above, we're not saying it is, we're just pointing to the other variable.

Ok. Will change the message.